styles/globalStyles.jsx

```diff
@@ -6,6 +6,11 @@ const GlobalStyles = createGlobalStyle`
 --button-border-radius: 0.3rem;
 --button-height: 2.5rem;
 --button-width: 6.5rem;
+--card-height: 455.13px;
```

Comment:
maybe insert comments for Structure to keep a better overview as the project grows` eg  // Card Styling`